deployment/test/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+cd "$(dirname "$0")"
+
+set -ex
+
+pid=
+cleanup() {
+    kill $pid
+    tilt down
+}
+trap cleanup EXIT
+
+tilt ci "$@"
+kubectl wait deployment/html --for=condition=available
```

Comment:
fwiw, i think this reveals a gap in the api - most people would want to attach a readiness probe to the deployment. `tilt ci` will wait for the readiness probe to pass.

Reply:
i might even do a 

```
local_resource('verify', 'curl -sf http://localhost:8008 | grep "Test deployment_create"', resource_deps['html'])
```

if you really want to make it tilt-native

Reply:
Yeah, that's probably better to do it in the Tiltfile, then I don't have to mess with port-forwarding in the test script.

I do like the idea of enabling readiness probes. Might play with a version that accepts arbitrary kwargs and patches them into the container.